great_expectations/cli/v012/datasource.py

```diff
@@ -1447,7 +1447,7 @@ def profile_datasource(
     open_docs=False,
     skip_prompt_flag=False,
 ):
-    """"Profile a named datasource using the specified context"""
+    """ "Profile a named datasource using the specified context"""
```

Comment:
I actually think this has an extraneous `"` right before `Profile`. Obviously something that `black` won't catch but would you mind just deleting that?